lib/rules/id-length.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @fileoverview Rule that warns when identifier names are shorter or longer
+ *               than the values provided in configuration.
+ * @author Burak Yigit Kaya aka BYK
```

Comment:
Those rules by me were created way before we started adding copyrights:-) But if you don't want to add one, that's perfectly fine.